A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -2942,24 +2947,37 @@ var A320_Neo_UpperECAM;
                 //Action
                 const action = document.createElement("span");
                 action.className = "Action";
-                let actionText = _item.action;
-                for (let i = 0; i < (19 - _item.name.length - _item.action.length); i++) {
-                    actionText = "." + actionText;
+                let actionText;
+                if (typeof _item.action === 'function') {
```

Comment:
Not super happy with this solution but I didn't want to do a large refactor in my first PR and my JS knowledge is rather weak. Happy to hear better ideas.